extensions/interactions/MultipleChoiceInput/protractor.js

```diff
@@ -19,16 +19,71 @@
 
 var forms = require(process.cwd() + '/core/tests/protractor_utils/forms.js');
 
+
 // The members of richTextInstructionsArray are functions, one for each option,
 // which will each be passed a 'handler' that they can use to edit the
 // rich-text area of the option, for example by
 //   handler.appendUnderlineText('emphasised');
-var customizeInteraction = function(elem, richTextInstructionsArray) {
-  forms.ListEditor(elem).setLength(richTextInstructionsArray.length);
-  for (var i = 0; i < richTextInstructionsArray.length; i++) {
-    var richTextEditor = forms.ListEditor(elem).editItem(i, 'RichText');
-    richTextEditor.clear();
-    richTextInstructionsArray[i](richTextEditor);
+// The second arguement(optional) editInstructions is used to edit the
```

Comment:
I think I already suggested not doing it like this. Pass things in as separate arguments rather than bundling them in "editInstructions".

If you like, you can confirm the docstring and function signature with me first before implementing it -- feel free to reply to this comment.

Reply:
See https://github.com/oppia/oppia/pull/8682#discussion_r402031683

Reply:
The function signature- `function(elem, richTextInstructionsArray, editCommand,  elementIndexToBeEdited)`
The doctring could go like: `
  The third and fourth arguments are optional and can be used to edit the interaction after it's creation. ,
  The third argument editCommand is a string('edit', 'delete', 'add') and the fourth argument  elementIndexToBeEdited is the index of the object to be edited or deleted. In the case of 'add' command,
  the fourth argument elementIndexToBeEdited is not  required.`

Reply:
This looks great. Thanks @donosco98!

Some nits:
- The fourth arg should be itemIndex, since: (a) you don't want to reuse "element" with a different meaning here, and (b) it's not always the case that this is being "edited".
- it's creation --> its creation (note that "it's" = "it is")
- You have an extra comma after "creation".

Reply:
done